website/components/LocationMap.vue

```diff
@@ -81,9 +81,18 @@ export default {
     mapIconSquareImg.src = mapIconSquare;
 
     this.map.on("load", () => {
+      const stateData = this.$store.state.usStates.usState;
+      stateData.features.splice(
+        0,
+        stateData.features.length,
+        ...stateData.features.filter(
+          (feature) => feature.geometry.coordinates[0] != null
+        )
```

Comment:
Is there a way we can ensure the `stateData` on the store's end is clean from the store's level? It may not be helpful to have null locations in general (think of another component that uses `this.$store.state.usStates.usState;`)

Reply:
Totally agree with you. I don't have a great idea of the project layout as I just cloned and started working on it an hour ago but I figured that list is useful with nulls because it also informs the list on the side (from what I can tell). So based on that I would imagine we still want those items to show up in the list view, just not on the map if they don't have coordinates.

Reply:
Actually I just realized this would break the list since it's modifying the data directly. Whoops. @Maker-Mark let me know (if you can) if my hunch was right and if so I'll change this so it uses a copied version for the map.

Reply:
I have a deep copy of this code ready to go. So if we replace the stateData assignment with 
```js
JSON.parse(
        JSON.stringify(this.$store.state.usStates.usState)
      );
```
we'll have a deep copy which keeps the unmappable items in the list but not on the map.

Reply:
I'm trying to figure out where the data is being fetched from. If we can sanitize it there, we should be able to get rid of them

Reply:
If we just want to fix it for locations. We can just filter the list instead of splicing the data. Filter will return a new array.

Reply:
> I'm trying to figure out where the data is being fetched from. If we can sanitize it there, we should be able to get rid of them

Sorry just saw this. It comes from the API directly to my knowledge
https://github.com/GUI/covid-vaccine-spotter/blob/3c8e7f255cde7ef634fbfb296ca0df5d3a505d32/website/pages/_state.vue#L147

Reply:
> If we just want to fix it for locations. We can just filter the list instead of splicing the data. Filter will return a new array.

Agreed. I think the issue there is that we also need to pass all the metadata along with it. We can just point to the same metadata since there's no issue with `store_count` being incorrect but since it's a negligible and we'd have to make that second object anyways I figured the deep copy is worth it.

Reply:
Just noticed this discussion after I started working on https://github.com/GUI/covid-vaccine-spotter/pull/141. I believe mapbox does not rely on metadata and I could possibly add a null check to the filteredLocationData, would that help?